kratos/tests/test_matrix_interface.py

```diff
@@ -1,51 +1,70 @@
 ﻿from __future__ import print_function, absolute_import, division
 
 import KratosMultiphysics.KratosUnittest as KratosUnittest
-from KratosMultiphysics import *
+import KratosMultiphysics as KM
 import math
 
-
 class TestMatrixInterface(KratosUnittest.TestCase):
 
+    def test_copy(self):
+        A = KM.Matrix(2,3)
+        A.fill(1.0)
+
+        b = KM.Vector(3)
+        b[0] = 1.0
+        b[1] = 2.0
+        b[2] = 3.0
+
+        A2 = A.copy()
+        b2 = b.copy()
+
+        self.assertEqual(2,A2.Size1())
+        self.assertEqual(3,A2.Size2())
+
+        for j in range(A2.Size2()):
+            self.assertEqual(b2[j], b2[j])
+            for i in range(A2.Size1()):
+                self.assertEqual(A[i,j], A2[i,j])

```

Comment:
could you please also check that the two objects are not the same one by checking the if the content is different after modifying one of them?
Thus you would make sure that not the same memory is referenced